### DISCUSSION TITLE
Return results with a title that matches the search term first

### DISCUSSION THREAD

**Discussion by hirasso** [Pagefind/pagefind#304] [Q&A]
Hey there! 

I have set up pagefind without customizing the sorting option. I would expect that if my search term matches a title of a result, this result should be displayed on the very top of the results list. This is not the case right now. Are there any special measures I would need to take to make this work?

Please tell me if you need more information.

Best,
Rasso

**Comment by bglw** [comment]
Hey @hirasso — this is indeed a blind spot in the current Pagefind model. At the moment the search result ranking is determined, rich details like the title and metadata has not yet been fetched, so can't currently influence the ranking.

The goal is to fix this with #13, where the title can be (automatically or manually) tagged with a weight, which will help boost those pages to better positions in the ranking. This is soon on my plate and a requirement for a Pagefind 1.0 release, which I'm hoping to land around July, so keep an eye on that 🙂 I'll try to remember to ping here if this makes it into an alpha/beta release early, too.
